### PR TITLE
Add Python dependency to AUGUSTUS-3.4.0-foss-2020b.eb

### PR DESCRIPTION
**Pull request by krometis**
Per https://github.com/easybuilders/easybuild-easyconfigs/issues/12660

### CODE REVIEW

**Comment by Micket** [comment]
Would you mind including the 2020a version as well?
It needs to also include a verionsuffix for Python like all the stuff did in 2020a.

**Comment by krometis** [comment]
It looks like I screwed this up:
1. I correctly submitted a PR for AUGUSTUS
2. I then got full of myself and decided I'd also submit a PR for a newer OpenMM build that I had lying around
3. I screwed up (2) and somehow added it to (1)

I don't know how to untangle the two now. I'm sorry about this...

**Comment by Micket** [comment]
So, alternatives in my opinion are
1. Just add Python as a build dep to satisfy the test-suite
2. Add Python and Perl as deps and also add `modextrapaths = {'PATH': 'scripts'}` (otherwise these deps don't do much at runtime anyway)

**Review by Micket — CHANGES_REQUESTED** [review]
Please also add Perl as a dep.

File: easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2020b.eb [hunk 1]
```diff
@@ -32,6 +32,8 @@ dependencies = [
     ('SuiteSparse', '5.8.1', '-METIS-5.1.0'),
     ('BamTools', '2.5.1'),
     ('SQLite', '3.33.0'),
+    ('Python', '3.8.6'),
+    ('Perl','5.32.0'),
```
Comment (Micket) [inline easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2020b.eb:36]:
```suggestion
    ('Perl', '5.32.0'),
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Going in, thanks @krometis!

**Pull request merged**